pyiron_base/database/generic.py

```diff
@@ -463,6 +472,20 @@ def _check_chem_formula_length(self, par_dict):
             par_dict[key_limited] = "OVERFLOW_ERROR"
         return par_dict
 
+    def add(self, obj, settings, target_manager: Type[TableManager]):
```

Comment:
As far as I know, setting object does not change in job, therefore I think we can instantiate an instance of setting in the `__init__`of the TableManager class.

Reply:
Agreed, but unfortunately I tried this and ran into circular import hell